web/locales/en/plugin__netobserv-plugin.json

```diff
@@ -453,6 +452,7 @@
   "Specify a single DNS RCODE name like:": "Specify a single DNS RCODE name like:",
   "A IANA RCODE number like 0, 3, 9": "A IANA RCODE number like 0, 3, 9",
   "A IANA RCODE name like NoError, NXDomain, NotAuth": "A IANA RCODE name like NoError, NXDomain, NotAuth",
+  "P": "P",
```

Comment:
what is `P` ?

Reply:
That's an autogenerated file :smiley: 

However `i18n` is a bit dumb and sort translations by first found in sources.
'P' is still used in Overview page since we don't have room to write 'Packets' inside donuts so that's why it's still there but in another order

Reply:
/lgtm